docs/en/04_Changelogs/4.6.0.md

```diff
@@ -75,7 +77,29 @@ module, you should upgrade it as well.
 
 Read [Allowed file types](Developer_Guides/Files/Allowed_file_types) in the Silverstripe CMS doc for all the details.
 
+## Simplify customisation of ModelAdmin {#modeladmin-customisation}
+
+`ModelAdmin::getEditForm()` has been split into smaller more discrete protected methods:
+* `getGridField()`
+* `getGridFieldConfig()`.
+
+Two matching extension hooks have been added as well:
+* `updateGridField()`
+* `updateGridFieldConfig()`.
+
+This will make it easier for developers to customise GridFields in their ModelAdmins.
+
+Learn how to [Alter the ModelAdmin GridField or Form](/developer_guides/customising_the_admin_interface/modeladmin/#altering-the-modeladmin-gridfield-or-form)
+
+## Login forms module ships with installer {#loginforms}
+
+The `silverstripe/login-forms` module is now part of the default installer. This alters the login form to provide
+consistent styling and behaviour that's independent from the specifics in your project. Only new project will
```

Comment:
```suggestion
consistent styling and behaviour that's independent from the specifics in your project. Only new projects will
```

Reply:
@maxime-rainville could you commit this suggestion (I'm unable to do this myself), I'll merge once you've done this

Reply:
Humm ... I would have assumed you should be able to do it. Is the "commit suggestion" button disabled for you?